doctrine/doctrine-bundle/2.4/config/packages/doctrine.yaml

```diff
@@ -12,7 +12,7 @@ doctrine:
         mappings:
             App:
                 is_bundle: false
-                type: annotation
+                type: annotation # use 'attribute' when the AttributeReader should be used
```

Comment:
its not an `AttributeReader` but `AttributeDriver`. I believe the sentence should be: `Use 'attribute' to switch to PHP 8 attributes based metadata mapping`